pkg/kube/apisix/apis/config/v2beta3/types.go

```diff
@@ -161,6 +161,8 @@ type ApisixRouteHTTPPlugin struct {
 	Enable bool `json:"enable" yaml:"enable"`
 	// Plugin configuration.
 	Config ApisixRouteHTTPPluginConfig `json:"config" yaml:"config"`
+	// Plugin configuration secretRef.
+	SecretConfig string `json:"secretConfig" yaml:"secretConfig"`
```

Comment:
Are we still adding features for v2beta3? @tao12345666333

Reply:
no, we don't need. https://github.com/apache/apisix-ingress-controller/pull/1486#discussion_r1039082058